rio_cogeo/cogeo.py

```diff
@@ -92,7 +92,7 @@ def cog_translate(  # noqa: C901
     allow_intermediate_compression: bool = False,
     forward_band_tags: bool = False,
     forward_ns_tags: bool = False,
-    quiet: bool = False,
+    quiet: Union[bool, TextIO] = False,
```

Comment:
I don't think we should use `quiet` for both.

we can add another option (name tbd)

Reply:
We can keep quite as it was before and add a new option maybe called `progress` or `progress_out`. I image the logic being like this...

```python
fout = ctx.enter_context(open(os.devnull, "w")) if quiet else sys.stderr
if quiet is False and progress_out:
    fout = progress_out
```

Reply:
Implemented. Ready for review.